game-core/src/main/java/games/strategy/triplea/delegate/PlaceDelegate.java

```diff
@@ -19,12 +16,4 @@
  * </p>
  */
 public class PlaceDelegate extends AbstractPlaceDelegate {
```

Comment:
As a side note:
I'm pretty sure `AbstractPlaceDelegate` can now be merged with `PlaveDelegate`